wscript

```diff
@@ -432,7 +432,9 @@ audio_output_features = [
     }, {
         'name': '--oss-audio',
         'desc': 'OSSv4 audio output',
-        'func': check_statement(['sys/soundcard.h'], 'int x = SNDCTL_DSP_SETPLAYVOL'),
+        'func': check_statement(['sys/soundcard.h'],
+                                'int x = SNDCTL_DSP_SETPLAYVOL; '
+                                'const char *mixer_path = PATH_DEV_MIXER'),
```

Comment:
`PATH_DEV_MIXER` [doesn't exist on FreeBSD](https://github.com/freebsd/freebsd-src/search?q=PATH_DEV_MIXER), so it cannot be used for auto-detecting OSSv4.

Reply:
Given `SOUND_MIXER_INFO` is defined on [Solaris](https://github.com/illumos/illumos-gate/search?q=SOUND_MIXER_INFO) and [Linux OSSv4](https://github.com/Open-Sound-System/OpenSoundSystem/search?q=SOUND_MIXER_INFO) the unused fallback code can be dropped by replacing `#elif defined(SOUND_MIXER_INFO)` with `#else`.
https://github.com/mpv-player/mpv/blob/171627b267bb741fafe0d06bf4dc9c6b9fe4571d/audio/out/ao_oss.c#L123-L124